README.rst

```diff
@@ -118,6 +118,17 @@ Contributing
 We value feedback and contributions from our community. Whether it's a bug report, new feature, correction, or additional documentation, we welcome your issues and pull requests. Please read through this `CONTRIBUTING <https://github.com/boto/boto3/blob/develop/CONTRIBUTING.rst>`__ document before submitting any issues or pull requests to ensure we have all the necessary information to effectively respond to your contribution.
 
 
+Maintenance and Support for SDK Major Versions
```

Comment:
I think we want this as a primary heading instead of a sub heading. We'll need dashes instead of tildas for that.

Reply:
Fixed